.gitignore

```diff
@@ -1,15 +1,12 @@
+*.egg-info
```

Comment:
This file's entries should be sorted.

Reply:
@mcdonnnj That file is already sorted.

```
$ cat .gitignore
*.egg-info
.DS_Store
.coverage
.env
.mypy_cache
.pytest_cache
.python-version
.venv/
.vscode
/Lib
__pycache__
uploads/
$ cat .gitignore | sort
*.egg-info
.DS_Store
.coverage
.env
.mypy_cache
.pytest_cache
.python-version
.venv/
.vscode
/Lib
__pycache__
uploads/
```

Reply:
Ah, we usually do in-editor sorting I believe and that provides a different result:

```
__pycache__
.coverage
.DS_Store
.env
.mypy_cache
.pytest_cache
.python-version
.venv/
.vscode
*.egg-info
/Lib
uploads/
```